core/ustring.cpp

```diff
@@ -28,6 +28,10 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Can you add a comment explaining why we define this? So that we can reassess in the future if we decide to actually use `strcpy_s` from C11.

Reply:
Done. 

"to disable warning which suggested to use strcpy_s instead strcpy"

Is it correct?